src/scripts/fetch/testing/fetch.test.ts

```diff
@@ -0,0 +1,120 @@
+import { APIFetch } from "../fetch";
+
+describe("should fetch data correctly.", () => {
+	test("should fetch movie details", async () => {
+		if (!process.env.APIKEY) {
+			console.log(process.env);
+			throw new Error(`MISSING API KEY! Data: ${process.env.APIKEY}`);
+		}
+		const data = await APIFetch("/movie/550", process.env.APIKEY);

```

Comment:
Could we use msw to mock api call otherwise it will keep calling original endpoint multiple times whenever test is run.

Reply:
Hey Mrunal, the mock server is run in setupTests.TS which is run before ALL jest Tests.
Code for the mock server is in the fetch/testing/handlers.ts and fetch/testing/server.ts

Reply:
Yes, checked now. Looks good.